README.md

```diff
@@ -187,4 +187,7 @@ request dated after any votes resets those votes.
 If the rule-change is adopted, a player who can shall merge the pull-request
 in a timely fashion, which marks completion of the vote.
 
-
+**301** If a PR that outlines a rule-change (in accordance with other rules in
+this file) contains `y` commit hashes containing an English dictionary word of 
```

Comment:
Disagree, why would we exclude rule repeals? I do agree with the general rephrasing that makes in independent of the rules that specify that rule-changes should be PRs. I would suggest just removing the part between parentheses.